src/getPoolColdPublicKey.c

```diff
@@ -0,0 +1,110 @@
+#ifdef POOL_OPERATOR_APP
```

Comment:
general note - given that cold keys are indeed the same as standard public keys (perhaps only with a different policy) - I would consider discarding this dedicated call and merging it with getExtendedPublicKey, only adding a policy to display the cold key when deriving even during the bulk key export